tests/table.html

```diff
@@ -0,0 +1,9 @@
+<table>
+<tr><th>1st field </th><th>2nd field </th><th>3rd field </th></tr>
```

Comment:
The headers don't follow alignment. Most other coverters [seems to align the header as well](https://babelmark.github.io/?text=%7C+1st+field+%7C+2nd+field+%7C+3rd+field+%7C%0A%7C+%3A--+++++++%7C+++%3A--%3A++++%7C+++++++--%3A+%7C%0A%7C+1st+entry+%7C+2nd+entry+%7C+3rd+entry+%7C%0A).

Reply:
Should be taken care of by https://github.com/karlb/smu/pull/11/commits/4e79b48498e2582fb0bf4eba442009417468d8aa